olive/workflows/run/config.py

```diff
@@ -34,6 +34,7 @@ class RunEngineConfig(EngineConfig):
     packaging_config: PackagingConfig = None
     log_severity_level: int = 1
     ort_log_severity_level: int = 3
+    tmp_dir_root: str = None  # if None, will use the whatever is set in OLIVE_TMP_DIR_ROOT
```

Comment:
should this belong engine config or run config? BTW, need the parameter passed by command line like:
python -m olive.workflows.run --config <config_path> --temp_dir blalblabla?